koku/api/report/queries.py

```diff
@@ -1358,6 +1361,11 @@ def _perform_rank_summation(self, entry, is_offset):
                 other[group] = others_label
             if 'account' in group_by:
                 other['account_alias'] = others_label
```

Comment:
We should check with Dan if the cards on the front end still require the `cluster_alias` to exist. We could do what we do for AWS here and just swap out the value with the others_label.

Reply:
Regardless, the "other" bucket shouldn't have those details for all cases because there's no guarantee that all of the "others" are all from the same cluster (or account).

Reply:
@dlabrecq - Can you weigh in on this?

Reply:
@dlabrecq said offline that the UI doesn't have a dependency on these fields. So, we're clear to remove them entirely.

Reply:
That's correct. The UI displays `cluster_alias` in the details page, when grouped by project or node, but we're not using limit there. In that case, we don't have 'others', so the change is safe.

Reply:
I did a test with a group_by[cluster] with multiple clusters and because we delete cluster every time, the field that says how many others there are gets deleted. Our backup for fields that don't have a name then takes over and fills in cluster with `no-cluster`. So I think we need to not delete cluster if it is the group by. 

```
GET /local/v1/reports/openshift/compute/?filter[time_scope_value]=-2&group_by[cluster]=*&filter[limit]=1
HTTP 200 OK
Allow: OPTIONS, GET
Content-Type: application/json
Vary: Accept

{
    "meta": {
        "count": 1,
        "group_by": {
            "cluster": [
                "*"
            ]
        },
        "filter": {
            "time_scope_value": "-2",
            "limit": 1,
            "time_scope_units": "month",
            "resolution": "monthly"
        },
        "total": {
            "usage": {
                "value": 2382.0,
                "units": "Core-Hours"
            },
            "request": {
                "value": 11910.0,
                "units": "Core-Hours"
            },
            "limit": {
                "value": 11910.0,
                "units": "Core-Hours"
            },
            "capacity": {
                "value": 9528.0,
                "units": "Core-Hours"
            },
            "infrastructure_cost": {
                "value": 0.0,
                "units": "USD"
            },
            "derived_cost": {
                "value": 0.0,
                "units": "USD"
            },
            "cost": {
                "value": 0.0,
                "units": "USD"
            }
        }
    },
    "links": {
        "first": "/local/v1/reports/openshift/compute/?filter%5Blimit%5D=1&filter%5Btime_scope_value%5D=-2&group_by%5Bcluster%5D=%2A&limit=100&offset=0",
        "next": null,
        "previous": null,
        "last": "/local/v1/reports/openshift/compute/?filter%5Blimit%5D=1&filter%5Btime_scope_value%5D=-2&group_by%5Bcluster%5D=%2A&limit=100&offset=0"
    },
    "data": [
        {
            "date": "2019-03",
            "clusters": [
                {
                    "cluster": "nise",
                    "values": [
                        {
                            "date": "2019-03",
                            "cluster": "nise",
                            "usage": {
                                "value": 1488.0,
                                "units": "Core-Hours"
                            },
                            "request": {
                                "value": 7440.0,
                                "units": "Core-Hours"
                            },
                            "limit": {
                                "value": 7440.0,
                                "units": "Core-Hours"
                            },
                            "capacity": {
                                "value": 5952.0,
                                "units": "Core-Hours"
                            },
                            "infrastructure_cost": {
                                "value": 0.0,
                                "units": "USD"
                            },
                            "derived_cost": {
                                "value": 0.0,
                                "units": "USD"
                            },
                            "cost": {
                                "value": 0.0,
                                "units": "USD"
                            },
                            "cluster_alias": "OCP Test Provider"
                        }
                    ]
                },
                {
                    "cluster": "no-cluster",
                    "values": [
                        {
                            "date": "2019-03",
                            "usage": {
                                "value": 894.0,
                                "units": "Core-Hours"
                            },
                            "request": {
                                "value": 4470.0,
                                "units": "Core-Hours"
                            },
                            "limit": {
                                "value": 4470.0,
                                "units": "Core-Hours"
                            },
                            "capacity": {
                                "value": 9528.0,
                                "units": "Core-Hours"
                            },
                            "infrastructure_cost": {
                                "value": 0,
                                "units": "USD"
                            },
                            "derived_cost": {
                                "value": 0,
                                "units": "USD"
                            },
                            "cost": {
                                "value": 0,
                                "units": "USD"
                            },
                            "cluster": "no-cluster"
                        }
                    ]
                }
            ]
        }
    ]
}
```